### DISCUSSION TITLE
Getting a common definition of 'Moral agency'

### DISCUSSION THREAD

**Discussion by matthewmccarthy11** [life-itself/community#700] [Research]
Hey everyone, 

After Friday's meeting a few of us had a brief discussion on what we mean by 'moral agency', and how to consider that term whenever we use it. Since 'agency' is a term that isn't easily defined, and neither really is 'moral' (don't hesitate to say if/why you disagree both of those claims), I wanted to talk more about how we might consider a concrete definition in our group (if such a term makes sense to use, and 
I think it does). A quick point: in the slides attached it sometimes says '(complex) moral agency' and as Simon pointed out 'complex' might not be necessary to distinguish, so for now the points just refer to 'moral agency'. Here are a few considerations as far as this goes:

1- That 'moral agency' broadly speaking is about 'enabling the good life' ('the good life' is not defined) 

2- That 'moral agency' is **not merely** an individual 'capacity' or 'process', but a relational 'capacity' or 'process' between people, the institutions and communities they are part of- and whatever else might be relevant in considering how 'the good life' can be enabled

3- How moral agency is or isn't different from moral development: my intuition is that the two terms can be coupled together, that 'moral agency' necessarily requires moral development and might always be in some kind of 'development', Simon thinks that the two terms are different and that we should use them separately

4- Some relevant concepts to get a common definition of 'moral agency' together are outlined in the slides, and others were suggested like the Cynefin Framework from Dave Snowden, and Ken Wilbers ideas on abstraction. Feel free to add others, or to add if I missed any

It was emphasized that in this ongoing conversation we need to be open to a variety of different 'moral development models' for different contexts etc. and that a singular concept like 'moral agency' needs to be a part of a wider web of concepts/models/frameworks etc.

Lastly Simon pointed out that 'the healing narrative' might be a good way to frame our discussion, which, correct me if I'm wrong, is something like: Why is life not great, how can we make it better? Perhaps this can be outlined on another discussion post later

Slides: 
[BrownBagJuly28.pdf](https://github.com/life-itself/community/files/12243243/BrownBagJuly28.pdf)

**Comment by asimong** [comment]
An opportunity to clarify what I was trying to say... Naturally moral agency and moral development will easily link together. And in the link we can see the difference: to me, moral development can lead to more moral agency. However I see moral development as broader. It could just be leading to more moral awareness, or more subtle and nuanced moral codes, or greater ability to do moral reasoning or (less fortunately) more moral self-righteousness. Though it could be argued that the last one wouldn't actually count as development! 

The point, to me, is that moral agency is the quality of being able to act in a moral way, not just think in a moral way. Other kinds of moral development may help moral agency, but do not strictly imply it. To me, it's in the words: an “agent” is one who/that acts, isn't that self-evident? So there's a basic ontological difference between development (a process) and agency (a disposition?) They are linked, but you can clearly differentiate the meaning.

**Reply by matthewmccarthy11** [reply]
Sounds good, let me just put it out like this:

moral agency (1): the quality of being able to act in a moral way
moral agency (2): the enabling of the good life

There are maybe some differences, but I think the two generally get to the same point, and for now, differentiating between 'moral agency' and 'moral development' also seems to make sense

**Reply by asimong** [reply]
@matthewmccarthy11 I'd appreciate an explanation of moral agency (2) as I don't understand it at the moment. Leaving aside the acknowledged issue of what is a good life (we know the implications there), I don't understand what "the enabling" is. Who is enabling, and how? I think I might call (2) something like "moral infrastructure" and I can't see the connection at present.

**Reply by asimong** [reply]
maybe you are thinking "moral agency" along the lines of "travel agency" or "employment agency"? That would kind of make sense of sense (2), but it has the drawback that nothing that I have read in the philosophical literature uses "agency" in this kind of way. Sure, the word is commonly used in this way, but I guess that's not what moral philosophers mean.

**Reply by asimong** [reply]
Thinking more about meanings of “agency” — in the sense of a travel or employment agency, it seems to mean an organisation that acts on your behalf; to which you delegate your “agency” in the first sense. Thus, a “moral agency”, though seeming to me rather a strange concept, would be an organisation to which you explained your moral values, and which acted morally on your behalf, in situations in which it is not convenient for you to act yourself directly, maybe because of the complexity of the situations in which you are called to act. (I hope this is interesting … I do think it is really helpful to talk out the different connotations of a word that seems unclear!)

**Reply by asimong** [reply]
This idea of a moral agency could perhaps tie in with Matthew's definition (2)? If you delegated your decisions to a moral agency then it could plausibly act on your behalf towards something that you regard as “the good life”.  Does that make any sense at all?

**Reply by matthewmccarthy11** [reply]
What I mean when I say an agency is 'an enabler' is more in line with how Francis Heylighen talks about agency in this paper: https://cris.vub.be/ws/portalfiles/portal/79556854/Heylighen_RelationalAgency.pdf

I guess I can point out too that the paper provides a framework on its own, which I generally agree with and find to be meaningful, and this definition of 'agency' really makes most sense in this framework. I guess I can also say that this is the 'best definition' in the sense that I think it makes most sense, but that doesn't mean it's a conclusive definition 

Some things that pop up: 'an agency can be seen as the enabler of a bundle of actions that act on the present situation so as to drive it towards a restricted set of attractor states' and he also points out that 'agencies can be defined in this framework as necessary conditions for the occurrence of a reaction'

An agency then might be viewed as a (mostly) repeatable pattern (reaction network) which creates conditions which enable certain actions, and these actions then enable conditions, etc.  
An agency then really is also synonymous with 'system' (curious to get your thoughts on this) 

In a 'moral context' then, seems to me that there should be an added emphasis on the conditions which 'enable the good life'

**Reply by asimong** [reply]
Ah, thanks for that paper @matthewmccarthy11 that explains a lot. To me it's a confusing use of the term agency, but by page 13 of the paper I have managed to grasp what he is getting at. To me, it's a bit like an embodied version of what in mathematics you would call a “transformation”.

I wonder if we could have a research session first going over Heylighen's concept of agency, for which people like me who are unfamiliar can read that paper in advance; and then talking about the extent to which it makes sense to talk about moral agency with this meaning of agency?

**Reply by asimong** [reply]
And it is immediately apparent to me that one could see “good” community as having moral agency in Heylighen's sense. Still a bit of “ouch” for me here though — I would say we need a separate term and would rather not use the term “moral agency” in that sense. One more thought, though — I can't see that kind of moral agency as being unchanged in the process, as Heylighen's definition suggests.

**Comment by asimong** [comment]
Also, to follow on from Matthew around healing narratives … 

To me, the most helpful way of seeing a healing narrative is in three parts: (a) the starting state (not good); (b) the end state (some kind of health etc.) and (c) the way, the path, or the means, to get from (a) to (b). The examples of this that I personally see most clearly are from religions: just to use one that's not mine, naively I might characterise the Buddhist healing narrative as from (a) delusion / illusion to (b) enlightenment through (c) meditation etc. Please feel free to correct me on that. The Christian ones seem slightly different between Orthodox, Roman Catholic and Protestant sects. Think “original sin” for starters. (denied by Pelagianism!) This is of course also called "salvific narrative" — and I don't have a particular reference for this, I'll leave the reader to search the 2350 results (I see on Google) on that quoted term.

What people I mix with sometimes leave out, surprisingly, is a clear expression of (a). Yes, we are mostly in tune with the regenerated eco-paradise that we're aiming for, but how do we characterise where we are starting from now? Different viewpoints lead to different paths to get there, and there can (sadly) be a lot of argument over that — and in my informal intuitive analysis, this is because we don't recognise our different starting points, and the different “paths up the same mountain” that follow from those different starting points. Thus, the different healing narratives related to Marxism, feminism, racism, etc. etc. etc.

The point here is that it seems to me that moral development begs the question: what is the underlying healing narrative? If greater "moral agency" is the end point, what are the starting points, and what are the developmental trajectories?

**Reply by asimong** [reply]
Or: (a) diagnosis (b) prognosis (c) treatment / therapy / cure

**Comment by asimong** [comment]
Having grasped (in outline at least) Heylighen's concept of agency, let me propose two things

1. That rather than trying to overload “moral agency” we use a term such as “moral support system”.
2. That a moral support system will typically have a healing narrative as a core organising principle.

Happy to enter into dialogue around these, and of course to try to identify the qualities of an effective moral support system (and related healing narratives) that actually encourages people towards the “good life” as we are conceiving it, rather than a system that coerces people into behaviour that is not in line with the wellbeing of people and planet.

**Reply by asimong** [reply]
When I say “overloading” here I mean specifically adding an extra different definition which is substantially different from the normal way in which moral agency is currently understood, as attested by the Wikipedia article https://en.wikipedia.org/wiki/Moral_agency 
(and I can't help feeling a little irritated that Heylighen seems to wantonly overload the term "agency" by itself )

**Comment by asimong** [comment]
[Pathways-to-Liberation-Self-Assess.pdf](https://github.com/life-itself/community/files/12440340/Pathways-to-Liberation-Self-Assess.pdf)
I give this as an example of a self-assessment grid that I see as inadequate, because the descriptors are too value laden. It is immediately clear that the ones on the right are "better" and more highly valued than the ones on the left. This is liable to two kinds of inaccuracy in self-assessment. First, plain Dunning-Kruger overestimation of skills by those who know little and are overconfident. Second, (less documented, but obvious to me) underestimation by those who have less self-confidence.
